pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -105,7 +126,7 @@ def create_intervals_mapping(from_lower_bounds, to_lower_bounds):
 
 def compute_vaccination_ratios(
         age_group_list, vaccinations_table, vacc_column, region_column,
-        population, merge_2022=False):
+        population, merge_2022=True):
```

Comment:
Is this input necessary?

Reply:
`merge_2022=False` is not used. but I think it is still good to be able to compute the ratios without merging.